docs/guide/templates.md

```diff
@@ -234,7 +234,7 @@ In order to display the Fields' value the way you want, you can either manipulat
 
 #### Getting raw Field value
 
-As you have access to the Content item in the template, you can use [its public methods](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/Core/Repository/Values/Content/Content.php) to access all the information you need. You can also use the `ez_field_value` helper to get the [Field's value only](twig_functions_reference.md#ez_field_value). It will return the correct language if there are several, based on language priorities.
+As you have access to the Content item in the template, you can use [its public methods](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/Core/Repository/Values/Content/Content.php) to access all the information you need. You can also use the `ez_field_value` helper to get the [Field's value only](content_rendering/twig_function_reference/field_twig_functions.md#ez_field_value). It will return the correct language if there are several, based on language priorities.
```

Comment:
```suggestion
Because you have access to the Content item in the template, you can use [its public methods](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/Core/Repository/Values/Content/Content.php) to access all the information you need. 
You can also use the `ez_field_value` helper to get the [Field's value only](content_rendering/twig_function_reference/field_twig_functions.md#ez_field_value). 
It returns the correct language if there are several, based on language priorities.
```